stylesheets/stylesheet.css

```diff
@@ -0,0 +1,70 @@
+.box {
+  height: 20em;
+  width: 15em;
+  border: 0.1em solid grey;
+  padding-left: 0.3em;
+}
+
+.container {
+  height: 400px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+img {
+  max-width: 100%;
+  margin: auto;
+}
+
+h2 {
+  color: green;
+}
+
+.checkout {
+  background-color: green;
+  color: white;
+  font-size: 115%
+}
+
+.nav1 {
+  color: darkblue;
+  background-color: grey;
+  font-size: 150%;
+  text-decoration: underline;
+}
+
+.nav2 {
+  color: white;
+  font-size: 150%;
+  text-decoration: underline;
+}
+
+.first {

```

Comment:
These aren't great class names for the sections. Maybe try to find something more semantic.